src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -30,25 +32,71 @@ class Route
     private array $schemes;
 
     /**
-     * @param array<string|\Stringable> $requirements
-     * @param string[]|string           $methods
-     * @param string[]|string           $schemes
+     * @param string|array<string,string>|null $path      The route path (i.e. "/user/login")
+     * @param string|string[]                  $methods   The list of HTTP methods allowed by this route
+     * @param string|string[]                  $schemes   The list of schemes allowed by this route (i.e. "https")
+     * @param string|null                      $locale    The locale accepted by the route
+     * @param string|null                      $format    The format returned by the route (i.e. "json", "xml")
+     * @param bool|null                        $utf8      Whether the route accepts UTF-8 in its parameters
+     * @param bool|null                        $stateless Whether the route is defined as stateless or stateful (@see https://symfony.com/doc/current/routing.html#stateless-routes)
      */
     public function __construct(
         string|array $path = null,
+
+        /**
+         * The route name (i.e. "app_user_login").
+         */
         private ?string $name = null,
+
+        /**
+         * Requirements for the route
+         * attributes (@see https://symfony.com/doc/current/routing.html#parameters-validation).
+         *
+         * @var array<string|\Stringable>
+         */
         private array $requirements = [],
+
+        /**
+         * Other route configuration like its prefix (i.e. ['prefix' => '/api']).
+         *
+         * @var array<string, mixed>
+         */
         private array $options = [],
+
+        /**
+         * The default values for the route attributes and
+         * query parameters.
+         *
+         * @var array<array-key, mixed>
+         */
         private array $defaults = [],
+
+        /**
+         * A host for which the route will be matched.
+         */
         private ?string $host = null,
         array|string $methods = [],
         array|string $schemes = [],
+
+        /**
+         * An expression that must evaluate to true for the route
+         * to be matched (@see https://symfony.com/doc/current/routing.html#matching-expressions).
```

Comment:
To avoid inline `@see` if possible.
```suggestion
         * An expression that must evaluate to true for the route to be matched.
         * 
         * @see https://symfony.com/doc/current/routing.html#matching-expressions
```